src/templates/base/2019/base_chapter.html

```diff
@@ -134,7 +134,7 @@ <h2 id="authors">
                 <div class="social">
                   {% if authordata.twitter %}
                   <a class="twitter" href="https://twitter.com/{{ authordata.twitter }}" aria-labelledby="author-{{ author }}-twitter">
-                    <svg width="22" height="22" role="img" aria-hidden>
+                    <svg width="22" height="22" role="img">
```

Comment:
Do you know why there is `role="img"` here and not on other pictos?

Reply:
Can you give me an example where it's not set? Just did a search for `<svg` and always seems to be set to me?

It's not necessary for `<img>` elements as they are directly mapped to images, whereas `<svg>`'s may not be. Is that what you mean?